torch/_inductor/codegen/aot_inductor_interface.cpp

```diff
@@ -1,5 +1,6 @@
 #include <torch/csrc/inductor/aot_inductor_interface.h>
 #include <torch/csrc/inductor/aot_inductor_model_container.h>
+#include <torch/script.h>
```

Comment:
This doesn't work for fbcode, where we have to explicitly "copy" (actually link) a number of header files to the temporary folder of the buck target (https://fburl.com/code/7mun56dt). 

Including ```script.h``` would basically force us to recursively "copy" all the directly- or indirectly- included header files to the temporary folder mentioned above. It's probably not a good idea. After checking the content of ```csr/jit/serialization/import.h```, which provides ```torch::jit::load```, it seems to be non-trivial to only including ```import.h``` and all the included files from it. Given that, I am thinking perhaps we should consider to let the user code (i.e. where we load aot_model.so) to load the constants and then pass the constant tensors to the model container. 

cc @desertfire @wushirong

Reply:
Give Yang's finding, I wonder whether we should switch to some native approach for save/load the constant files. I saw c10::load is one option, but need to find the proper save method.

Reply:
Pulling @SherlockNoMad into discussion.

Reply:
Let's still bundle the weights into .so to make it self-contained without dependencies. We can still keep the file required for fallback.

Reply:
Chatted with @chenyang78 offline. 
Runtime should be responsible for loading the weight, we shouldn't need to load weight in .so.